### PR TITLE
Add 16bit norm and NV12 textures

### PR DESCRIPTION
**Pull request by davnotdev**
`Features::TEXTURE_FORMAT_16BIT_NORM` and `Features::TEXTURE_FORMAT_NV12` are currently exposed as native features. This PR adds a new enum `WGPUTextureFormatExtras` which will make the texture formats of the mentioned features accessible.

### CODE REVIEW

File: src/conv.rs [hunk 1]
```diff
@@ -819,6 +819,13 @@ pub fn map_texture_format(value: native::WGPUTextureFormat) -> Option<wgt::Textu
         native::WGPUTextureFormat_ASTC12x10UnormSrgb => Some(wgt::TextureFormat::Astc { block: AstcBlock::B12x10, channel: AstcChannel::UnormSrgb }),
         native::WGPUTextureFormat_ASTC12x12Unorm => Some(wgt::TextureFormat::Astc { block: AstcBlock::B12x12, channel: AstcChannel::Unorm }),
         native::WGPUTextureFormat_ASTC12x12UnormSrgb => Some(wgt::TextureFormat::Astc { block: AstcBlock::B12x12, channel: AstcChannel::UnormSrgb }),
+        native::WGPUTextureFormatExtras_R16Unorm => Some(wgt::TextureFormat::R16Unorm),
```
Comment (almarklein) [inline src/conv.rs]: Why not simply `WGPUTextureFormat_R16Unorm`?

Comment (almarklein) [reply src/conv.rs]: Oh, I guess because the prefix is the full enum name...

Comment (davnotdev) [reply src/conv.rs]: I'm a bit divided on the naming. I do believe in the importance of separating types that are outside the spec and following established precedent, but on the other hand, `WGPUTextureFormat_R16Unorm` feels much more natural and may become apart of the spec one day anyway. Let me which you would prefer.

Comment (almarklein) [reply src/conv.rs]: Right, from the point of view of wgpu-core the texture formats are equal, but from the pov of wgpu native they are not, so I'd prefer keeping the "extra" in the prefix. @rajveermalviya thoughts?

Comment (rajveermalviya) [reply src/conv.rs]: Yeah, there isn't much consistency in naming extended enums/structs in `wgpu.h`, we have enums like `WGPUNativeFeature` (extends `WGPUFeatureName`), and structs use the `Extras` suffix.

So I think we should have the `WGPUNative` prefix for extended enums here matching other extended enums, for consistency.

I'll make that change and then merge.

**Review by rajveermalviya — APPROVED** [review]
Thanks!

**Pull request merged**